logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/rootDetection/CheckSystemIntegrityUseCase.kt

```diff
@@ -43,17 +43,17 @@ interface CheckSystemIntegrityUseCase {
         object Failed : Result()
     }
 
-    suspend fun invoke(): Result
+    suspend operator fun invoke(): Result
 }
 
-class CheckSystemIntegrityUseCaseImpl(
+internal class CheckSystemIntegrityUseCaseImpl(
     private val kaliumConfigs: KaliumConfigs,
     private val rootDetector: RootDetector,
     private val sessionRepository: SessionRepository,
 ) : CheckSystemIntegrityUseCase {
 
     override suspend fun invoke(): CheckSystemIntegrityUseCase.Result {
-        return if (kaliumConfigs.wipeOnRootedDevice && rootDetector.isSystemRooted()) {
+        return if (true) {
```

Comment:
is it expected to have this hard-coded?

Reply:
noo srry :P

Reply:
I was about to push but you were faster :P